### PR TITLE
simple pr template

### PR DESCRIPTION
**Pull request by outofambit**
A lightweight-as-possible template to help pull requests get linked to their corresponding issue.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Overview
+
+<!-- Please insert what the number of the issue you are addressing. (If an issue doesn't exist for this PR to address, please open one at https://github.com/desktop/desktop/issues before opening this PR.) -->
```
Comment (billygriffin) [inline .github/pull_request_template.md]:
```suggestion
<!-- Please insert what the number of the issue you are addressing. (If an issue doesn't exist for this PR to address, please open one at https://github.com/desktop/desktop/issues/new/choose before opening this PR.) -->
```

Comment (billygriffin) [inline .github/pull_request_template.md]:
```suggestion
<!-- Please insert the number of the issue you are addressing. (If an issue doesn't exist for this PR to address, please open one at https://github.com/desktop/desktop/issues before opening this PR.) -->
```

Comment (billygriffin) [inline .github/pull_request_template.md]:
```suggestion
<!-- Please insert what the number of the issue you are addressing. (If an issue doesn't exist for this PR to address, please open one at https://github.com/desktop/desktop/issues to allow for discussion before opening this PR.) -->
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+**resolves #[issue number]**
```
Comment (iAmWillShepherd) [inline .github/pull_request_template.md]: Let's use simpler words.

```suggestion
**Fixes #[issue number]**
```

File: .github/pull_request_template.md (hunk 1 above)
Comment (iAmWillShepherd) [inline .github/pull_request_template.md]:
```suggestion
<!-- 
What issue are you addressing? e.g. #1234 
If there is no issue for this pull request (PR), please create one before opening your PR.
Issues can be created at https://github.com/desktop/desktop/issues) 
-->
```

I think ☝️ 
* Increases readability by separating content from markup
* Question makes clear what we're looking for
* Helps guide user by giving an example
* Defines acronym before using it

**Comment by outofambit** [comment]
@billygriffin @iAmWillShepherd wanna take another look or shall i #yolo it? #jolo #yolonely

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+## Overview
+
+<!-- Please insert what the number of the issue you are addressing. (If an issue doesn't exist for this PR to address, please open one at https://github.com/desktop/desktop/issues/new/choose before opening this PR.) -->
+**Fixes #[issue number]**
+
+-
```
Comment (billygriffin) [inline .github/pull_request_template.md]: Does the bullet here have a purpose? I might consider leaving that out?

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+## Overview
+
+<!-- Please insert what the number of the issue you are addressing. (If an issue doesn't exist for this PR to address, please open one at https://github.com/desktop/desktop/issues/new/choose before opening this PR.) -->
```
Comment (billygriffin) [inline .github/pull_request_template.md]:
```suggestion
<!-- Please insert the number of the issue you are addressing. (If an issue doesn't exist for this PR to address, please open one at https://github.com/desktop/desktop/issues/new/choose to allow for discussion before opening this PR.) -->
```

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+## Overview
+
+<!--
+What issue are you addressing? (for example, #1234)
+
+If an issue doesn't exist for this pull request (PR) to address, please open one to allow for discussion before opening this PR.
+(You can open a new issue at https://github.com/desktop/desktop/issues/new/choose)
+-->
+**Closes #{issue number}**
```
Comment (iAmWillShepherd) [inline .github/pull_request_template.md]: Will this formatting prevent GitHub from automagically linking this issue?

**Pull request merged**